Services/LegalDocuments/classes/ConsumerToolbox/ConsumerSlots/Agreement.php

```diff
@@ -61,17 +61,21 @@ public function showAgreement(string $gui, string $cmd): PageFragment
 
     public function showAgreementForm(string $gui, string $cmd): PageFragment
     {
+        $form = $this->user->matchingDocument()->isOk() ?
+              $this->agreementForm($gui, $cmd) :
+              $this->ui->create()->legacy('<br/><br/>');
```

Comment:
Can we replace this by a divider or something else from the UI framework?

Reply:
Ok.

Reply:
Is changed to a divider.